crates/vm/levm/src/vm.rs

```diff
@@ -133,23 +135,21 @@ impl VM {
 
                 default_touched_accounts.insert(new_contract_address);
 
-                // (3)
                 let created_contract = Account::new(value, calldata.clone(), 1, HashMap::new());
                 cache::insert_account(&mut cache, new_contract_address, created_contract);
 
-                // (5)
-                let code: Bytes = calldata.clone();
+                let bytecode: Bytes = calldata.clone();
 
                 let initial_call_frame = CallFrame::new(
                     env.origin,
                     new_contract_address,
                     new_contract_address,
-                    code,
+                    bytecode,
                     value,
-                    Bytes::new(),
```

Comment:
In a create tx calldata should be an empty set of bytes, not the calldata.

Reply:
Good catch! Didn't know that
Fixed it here:
https://github.com/lambdaclass/ethrex/pull/1345/commits/79fc4e080efffddd4519d476ce1efa9a57e3d77c